src/move/tasks/identify_associations.py

```diff
@@ -48,15 +48,24 @@ def _validate_task_config(
         if len(task_config.num_latent) != 4:
             raise ValueError("4 latent space dimensions required.")
 
-
 def identify_associations(config: MOVEConfig):
-    """Trains multiple models to identify associations between the dataset
-    of interest and the continuous datasets."""
+    """Leads to the execution of the appropiate association 
+    identification functions."""
+
+    task_config = cast(IdentifyAssociationsConfig, config.task)
+    if task_config.target_value in ContinuousTargetValue:
+        identify_associations_continuous(config)
+    else:
+        identify_associations_categorical(config)
```

Comment:
Looks like there's not many differences between the approach functions (only the mask). So I think, it would be better to do something like:

```python
# pseudo-code
def identify_associations(config):
    if continuous_target:
        dataloaders, mask, etc = prepare_for_continuous_perturbation(config)
    else:
        dataloaders, mask, etc = prepare_for_categorical_perturbation(config)
    def _bayes_approach(config):
        # do bayes
    def _ttest_approach(config):
        # do ttest
    if bayes:
        results = _bayes_approach(config)
    elif ttest:
        results = _ttest_approach(config)
    save_results(results)
```

This makes the code easier to maintain. If there's something we need to change about the ttest/bayes, we only have to change it once in the code, and not fear we forgot to duplicate the fix in the other function.